modules/sharethroughBidAdapter.js

```diff
@@ -1,123 +1,178 @@
 import { registerBidder } from '../src/adapters/bidderFactory.js';
 import * as utils from '../src/utils.js';
 import { config } from '../src/config.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { createEidsArray } from './userId/eids.js';
 
-const VERSION = '3.4.1';
+const VERSION = '4.0.0';
 const BIDDER_CODE = 'sharethrough';
-const STR_ENDPOINT = 'https://btlr.sharethrough.com/WYu2BXv1/v1';
-const DEFAULT_SIZE = [1, 1];
+const SUPPLY_ID = 'WYu2BXv1';
+
+const STR_ENDPOINT = `https://btlr.sharethrough.com/universal/v1?supply_id=${SUPPLY_ID}`;
 
 // this allows stubbing of utility function that is used internally by the sharethrough adapter
 export const sharethroughInternal = {
-  b64EncodeUnicode,
-  handleIframe,
-  isLockedInFrame,
-  getProtocol
+  getProtocol,
 };
 
 export const sharethroughAdapterSpec = {
   code: BIDDER_CODE,
+  supportedFormat: [BANNER, VIDEO],
```

Comment:
This isn't needed with `supportedMediaTypes` defined in the spec

Reply:
Ok, removing `supportedFormat` attribute here